compiler/src/iree/compiler/Codegen/Common/test/decompose_softmax_local_fusion_false.mlir

```diff
@@ -0,0 +1,46 @@
+// RUN: iree-opt --split-input-file --iree-codegen-decompose-softmax=use-fusion=false -cse %s | FileCheck %s
```

Comment:
Instead of separate files, you can just check the different flag values with `--check_prefix=<LABEL>` and combine everything into a single file.

Reply:
Done
Thanks!